src/postamble.js

```diff
@@ -316,6 +316,10 @@ Module['exit'] = Module.exit = exit;
 var abortDecorators = [];
 
 function abort(what) {
+  if (Module['onRuntimeInitializeFailed']) {
```

Comment:
Yeah, I think that change is more correct regardless - we should call `abort` when forced to abort execution. (It's more important after execution starts, since abort makes sure we stop running events afterwards; but it's better for consistency to do it in all places.)